README.md

```diff
@@ -508,12 +508,16 @@ competition that contains tags for approx 14M publications from PubMed.
 
 ## 🐍 Venv
 
-To create and setup the development environment
+To create and setup the base environment
```

Comment:
You need to move the commands around and introduce a new header development as described in the issue. The user needs to know which commands work out of the box compared to what needs additional installation